A33-Fault-Injection.md

```diff
@@ -117,6 +117,11 @@ gRPC should accept fault injection settings with either HTTP status code or gRPC
 According to the [Envoy implementation](https://github.com/envoyproxy/envoy/blob/main/source/extensions/filters/http/fault/fault_filter.cc#L222), each feature of fault injection is checked with a different random number. Compared to using a single random number for all features, this affects the probability of an RPC being affected by fault injection. For example, the fault injection has 20% chance to delay and 5% chance to abort. Under the single random number solution, only 20% RPC will be affected. But if two features are independent, 24% RPC will be affected. So, **this doc suggests to make each feature independent by using a different random number each time.**
 
 
+### Environment Variable Protection
```

Comment:
Please make changes similar to what I did in https://github.com/grpc/proposal/pull/219/commits/af7b012a38e677f5721f1b021b2e829809802aa9 to reflect the fact that this is just temporary.

Reply:
Done. Using the same words as your commit.